### DISCUSSION TITLE
mdfind doesnt work with consult

### DISCUSSION THREAD

**Discussion by dasoju** [minad/consult#1129] [Q&A]
I looked at all the old posts and set 
(setq consult-locate-args "mdfind -name")

consult-locate #xxx 
does not generate any results

when i use mdfind -name xxx
on the macos terminal i get the list of files with *xxx*

**Comment by minad** [comment]
`mdfind` is not officially supported, but you can check the ` *consult-async*` buffer for the executed commands and errors and compare with and without your customization.
